src/Symfony/Bundle/SecurityBundle/Tests/Functional/SetAclCommandTest.php

```diff
@@ -54,6 +54,9 @@ protected function tearDown()
         $this->deleteTmpDir('Acl');
     }
 
+    /**
+     * @group legacy
```

Comment:
This is necessary until DoctrineBundle is compatible with the changes done in this PR.